application/modules/layout/views/includes/head.php

```diff
@@ -32,6 +32,10 @@
     <script src="<?php echo base_url(); ?>assets/core/js/locales/select2/<?php _trans('cldr'); ?>.js"></script>
 <?php } ?>
 
+
+<script src="<?php echo base_url(); ?>assets/core/js/bootstrap-markdown.js"></script>
```

Comment:
Using the OS bootstrap markdown plugin:
https://github.com/toopay/bootstrap-markdown